www/push.js

```diff
@@ -1,8 +1,8 @@
 /**
 * This file has been generated by Babel.
-* 
```

Comment:
Since the only updates to this file are whitespace you can omit this file from your PR.

Reply:
Ok